extensions/sql-bindings/src/common/azureFunctionsUtils.ts

```diff
@@ -618,35 +638,36 @@ export async function getConnectionURI(connectionInfo: IConnectionInfo): Promise
 	return connectionURI;
 }
 
-export async function promptSelectTable(connectionURI: string, bindingType: BindingType, selectedDatabase: string): Promise<string | undefined> {
+export async function promptSelectObject(connectionURI: string, bindingType: BindingType, selectedDatabase: string, isView: boolean): Promise<string | undefined> {
```

Comment:
You can pass in the objectType enum here that @Charles-Gagnon suggested instead of the isView boolean

Reply:
I will still have to use a isView boolean to determine certain prompts within promptSelectObject. Not exactly sure if just sending the enum would solve the case where user selects a table and then we still have to see if it is a view.

Reply:
```
if(objectType == ObjectType.View) {
   // Do view-specific code here
}
```

Reply:
So I can move the line in [366](https://github.com/microsoft/azuredatastudio/pull/19894/files#diff-942c301818b2962d582b470ff15d00da98787e95e3bc98a95f79de6bae4be102R366) into promptSelectObject. Most of the view-specific code is done via ternary operations with the function.

Reply:
Yeah. The advantage of this is making it so that you can do other operations for other types without needing to change the signature any more. Keeping things generic is generally better.